app/assets/stylesheets/shared/select.scss

```diff
@@ -0,0 +1,73 @@
+.sn-select {
+  cursor: pointer;
+  position: relative;
+  padding: .5em;
+  border: 1px solid;
+  border-radius: 4px;
+  border-color: var(--sn-grey);
+
+  .sn-select__value {
+    all: unset;
+  }
+
+  &:hover {
+    background: var(--sn-sleepy-grey);
+  }
+
+  .sn-select__options {
+    display: none;
+  }
+
+  .sn-select__caret {
+    position: absolute;
+    right: 1em;
+    top: 1.1em;
```

Comment:
you can just define the parent container with the same height as select and center caret inside. Then you don't need this calculation.